src/main/webapp/css/main.css

```diff
@@ -164,7 +164,3 @@ h1 {
   background-color: #f88379;
   color: white;
 }
-
-.reviewer {
-  color: #f88379;
```

Comment:
I actually use this in reviewer-loader.js so pls don't delete it haha

Reply:
whoops, my bad! It is put back :)